### DISCUSSION TITLE
Admin dashboard vs themes

### DISCUSSION THREAD

**Discussion by brez7** [twbs/bootstrap#39318] [Q&A]
Does the admin and dashboard integrate with themes? Similar to Wordpress?

**Comment by monab** [comment]
Hi @brez7,

Are you inquiring whether it is possible to install admin dashboards & themes, such as themes installed in WordPress?

In that case, no, these themes are not designed for use as WordPress themes. They are, in fact, Bootstrap admin dashboards or themes intended for backend user interface (UI) purposes. It's important to note that WordPress itself serves as a backend system with its own UI.
